src/components/__tests__/RouteListEdit.cy.js

```diff
@@ -21,6 +32,7 @@ describe('<RouteListEdit>', () => {
 
   context('desktop', () => {
     beforeEach(() => {
+      cy.clock(new Date('2024-08-15').getTime());
```

Comment:
Date literal string repeat across code, you can use const instead of string literal.

Reply:
Refactored into a variable `fixedDate`.